JoyboyCommunity/src/components/TippingModal/styles.ts

```diff
@@ -0,0 +1,61 @@
+import {Spacing, ThemedStyleSheet} from '../../styles';
+
+export default ThemedStyleSheet((theme) => ({
+  container: {
+    flex: 1,
+  },
+  background: {
+    flex: 1,
+  },
+  backgroundImage: {
+    width: '100%',
+    height: '100%',
+  },
+  modalContainer: {
+    position: 'absolute',
+    bottom: 0,
+    left: 0,
+    right: 0,
+    width: '100%',
+    height: '100%',
+    alignItems: 'center',
+    justifyContent: 'center',
+  },
+  modal: {
+    width: '80%',
+    height: 291,
+    backgroundColor: theme.colors.surface,
+    borderRadius: 24,
+    paddingVertical: Spacing.medium,
+    paddingHorizontal: Spacing.large,
+  },
+  logo: {
+    position: 'absolute',
+    top: '-43%',
+    left: 0,
+    right: 0,
+    width: '100%',
+    height: '100%',
+    alignItems: 'center',
+    justifyContent: 'center',
+  },
+  content: {
+    gap: 16,
```

Comment:
In the figma design, gap is 12, can we use 12 here too? The design should match the app 1:1